com.unity.shadergraph/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Math/Trigonometry/Arctangent2Node.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections.Generic;
+using static UnityEditor.ShaderGraph.Registry.Types.GraphType;
+
+namespace com.unity.shadergraph.defs
+{
+
+    internal class Arctangent2Node : IStandardNode
+    {
+        public static FunctionDescriptor FunctionDescriptor => new(
+            1,     // Version
+            "Arctangent2", // Name
+            "Out = atan2(A, B);",
+            new ParameterDescriptor("A", TYPE.Vector, Usage.In),
+            new ParameterDescriptor("B", TYPE.Vector, Usage.In),
+            new ParameterDescriptor("Out", TYPE.Vector, Usage.Out)
+        );
+
+        public static Dictionary<string, string> UIStrings => new()
+        {
+            { "Category", "Math, Trigonometry" },
+            { "Name.Synonyms", "atan2" },
+            { "Tooltip", "returns the arctangent of A divided by B" },
```

Comment:
Just some thoughts here, do we want to specify the differences of Arctangent and Arctangent2 nodes in the tooltips?

Reply:
So arctan2 does arctan(A/B) instead of arctan(A).  And the tool tip says that.  What would you like it to say instead?